src/app/modules/epidemiologia/components/ficha-epidemiologica-crud/ficha-epidemiologica-crud.component.ts

```diff
@@ -338,13 +338,15 @@ export class FichaEpidemiologicaCrudComponent implements OnInit, OnChanges {
         id: this.paciente.id,
         documento: this.paciente.documento,
         nombre: this.paciente.nombre,
+        nombreAutopercibido: this.paciente.alias ? this.paciente.alias : null,
```

Comment:
Debería ser lo mismo, null o algún valor

```suggestion
        alias:  this.paciente.alias,
```